stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/builder/components/ServiceStub.java

```diff
@@ -81,6 +86,10 @@ public Builder addMethod(Method method) {
             return this;
         }
 
+        public void setProxyAvailable() {
```

Comment:
Shall we rename this function as `enableProxy` or something like that? This name is confusing since you are just set it to true?

Reply:
I changed it to enableProxy.